command/secretsManager/secret_manager.go

```diff
@@ -0,0 +1,26 @@
+package secretsManager
```

Comment:
Don't use camelCase names for packages, it should either be `secret` or `secretmanager`

Reply:
Standardized the naming 👍 

8792149